test/base/SpecBase.scala

```diff
@@ -72,21 +72,33 @@ trait SpecBase extends PlaySpec with GuiceOneAppPerSuite with TryValues with Sca
      """.stripMargin
   )
 
-  lazy val validPaperlessPreferences: JsValue = Json.parse(
+  lazy val paperlessCustomer: JsValue = Json.parse(
     s"""
        |{
-       |  "termsAndConditions": {
-       |    "generic": {
-       |      "accepted": true
-       |    },
-       |    "taxCredits": {
-       |      "accepted": true
-       |    }
+       |  "status": {
+       |    "name": "ALRIGHT",
```

Comment:
This is really quite strange of them to give us.. Alright??

Reply:
I totally agree but that is the value they have used
![image](https://user-images.githubusercontent.com/11269301/94257529-56989580-ff23-11ea-9815-999c1a51400c.png)

Reply:
think about this in context: 
1. you are calling paperless/status endpoint to get a paperless status;
2. the possible return values are: https://github.com/hmrc/preferences-frontend/blob/master/app/model/StatusWithUrl.scala#L16
3. You have 4 paperless statuses which indicate a problem: PAPER, NEW_CUSTOMER, EMAIL_NOT_VERIFIED, EMAIL_BOUNCED;
4. You have 1 paperless status which indicates a customer is paperless with no problem: ALRIGHT;
5. Soon you will have 1 paperless status which will indicate a customer is paperless but it has a problem: OLD_VERSION, meaning he opted in with an old version of the terms and conditions and needs to RE_OPT_IN;

So in the above context, some clients, like mobile app want to have a fine grained status to display different screens/user journeys. For BTA they would want to show a specific message for the customers. PTA don't care as they just call paperless interrupt and we do the status calculation and displaying the page for them;
For other clients, like yourself, you would want to know just if the customer is paperless so that he can receive digital communication. For this use case we will have a field called isPaperless: Boolean. 
If you have any suggestion of a better alternative to ALRIGHT, we would be happy to hear it ;)

Reply:
at the time of writing **isPaperless: Boolean = true/false** is not currently available